test/core/client_channel/service_config_test.cc

```diff
@@ -154,6 +162,9 @@ class ErrorParser : public ServiceConfigParser::Parser {
   static const char* MethodError() { return "ErrorParser : methodError"; }
 
   static const char* GlobalError() { return "ErrorParser : globalError"; }
+
+ private:
+  absl::string_view name_;
 };
 
 class ServiceConfigTest : public ::testing::Test {
```

Comment:
Please also add a test for what happens when two parsers are registered with the same name.

Reply:
done